readme.md

```diff
@@ -176,6 +176,9 @@ Set the viewport dimensions
 #### .screenshot(path)
 Saves a screenshot of the current page to the specified `path`. Useful for debugging. Note the path must include the file extension. Supported formats include .png, .gif, .jpeg, and .pdf.
 
+#### .extractDomElement(selector,path)
```

Comment:
Yea, your right. Sorry English it's not my main language even though I'm very good at it, hehe ^_^. I'll exercise more when I come to your country and dance on the clubs tables.